core/lib/dal/src/consensus_dal.rs

```diff
@@ -317,7 +320,36 @@ impl ConsensusDal<'_, '_> {
         else {
             return Ok(None);
         };
-        Ok(Some(zksync_protobuf::serde::deserialize(row.certificate)?))
+        let Some(certificate) = row.certificate else {
+            return Ok(None);
+        };
+        Ok(Some(zksync_protobuf::serde::deserialize(certificate)?))
+    }
+
+    pub async fn batch_committee(
```

Comment:
It would be great to state it in docstrings that this returns the committee which should attest to the `batch_number`, and not the one which is the result of executing the batch, ie. the ones attesting for `batch_number+1`.